yjit/src/options.rs

```diff
@@ -87,13 +99,18 @@ pub static mut OPTIONS: Options = Options {
 };
 
 /// YJIT option descriptions for `ruby --help`.
-static YJIT_OPTIONS: [(&str, &str); 8] = [
+static YJIT_OPTIONS: [(&str, &str); 11] = [
     ("--yjit-stats",                    "Enable collecting YJIT statistics"),
     ("--yjit-trace-exits",              "Record Ruby source location when exiting from generated code"),
     ("--yjit-trace-exits-sample-rate",  "Trace exit locations only every Nth occurrence"),
     ("--yjit-exec-mem-size=num",        "Size of executable memory block in MiB (default: 128)"),
-    ("--yjit-call-threshold=num",       "Number of calls to trigger JIT (default: 30)"),
+
+    ("--yjit-call-threshold=num",       "Number of calls to trigger JIT"),
+    ("--yjit-small-threshold=num",      "Call threshold for small apps (default: 30)"),
+    ("--yjit-large-threshold=num",      "Call threshold for large deployments (default: 120)"),
```

Comment:
I think we agreed on not documenting these in `ruby --help`? I guess you changed your mind since it's hard to document the defaults without them?

Reply:
can't it just be hardcoded? If you pass an explicit `--call-threshold` then we ignore them.

Then these can be documented as part of the `--yjit-call_thresold` default.

Reply:
Yes you're right, we did agree, and I remembered it after opening the PR. I'll undocument these options 😆

Reply:
Jean's suggestion is not a bad one. I could define two constants instead.